### PR TITLE
AppVeyor

### LINKED ISSUE #117: AppVeyor Builder
**Feature description**
Add AppVeyor build/test option to build MacOS and Windows release on native OS, not on linux with GOOS build option of Go.

### PR DESCRIPTION
**Pull request by mrpalide**
Fixes #117 

Changes: 
- Add `.appveyor.yml` configuration file.

Does this change need to mentioned in CHANGELOG.md?
No

### CODE REVIEW

**Comment by mrpalide** [comment]
**Issues**:
- Windows: Failed on `pkg/prepconf` test. [`link`](https://ci.appveyor.com/project/mrpalide/skybian/build/job/ivp4kep01kc21ogg#L327)
- MacOS: Failed on `pkg/imager` test. [`link`](https://ci.appveyor.com/project/mrpalide/skybian/build/job/aqs35ujgfidn7teg#L70)

**Comment by asxtree** [comment]
Added the latest armbian image and changed the way to differentiate the skywire bins to download and for the chroot through architecture type and not board how it was till now.

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+version: "{build}"
+
+stack: go 1.16.7
+
+environment:
+  matrix:
+  # For regular jobs, such as push, pr and etc.
+  - job_name: Linux
+    appveyor_build_worker_image: ubuntu1804
+  # - job_name: MacOS
```
Comment (jdknives) [inline .appveyor.yml]: This never needs to run on Mac or Windos so we should just remove it imo.

File: pkg/prepconf/prepare.go [hunk 2]
```diff
@@ -104,19 +104,13 @@ func generateConfig(conf Config) (*visorconfig.V1, error) {
...
 	out.Hypervisors = bp.HypervisorPKs
 	out.LogLevel = skyenv.DefaultLogLevel
 	out.ShutdownTimeout = visorconfig.DefaultTimeout
 	out.RestartCheckDelay = visorconfig.Duration(restart.DefaultCheckDelay)
 	out.Launcher = &visorconfig.V1Launcher{
-		Discovery: &visorconfig.V1AppDisc{
-			ServiceDisc:    skyenv.DefaultServiceDiscAddr,
-			UpdateInterval: visorconfig.Duration(skyenv.AppDiscUpdateInterval),
-		},
+		ServiceDisc: skyenv.DefaultServiceDiscAddr,
```
Comment (jdknives) [inline pkg/prepconf/prepare.go:113]: There seem to be a few things missing that we currently support on `develop`. Just generate a config on develop and you will be able to compare the differences easily.

**Review by jdknives — CHANGES_REQUESTED** [review]

File: .appveyor.yml (hunk 1 above)
Comment (mrpalide) [reply .appveyor.yml]: Done.

File: pkg/prepconf/prepare.go (hunk 2 above)
Comment (mrpalide) [reply pkg/prepconf/prepare.go:113]: Done.

**Review by jdknives — APPROVED** [review]

**Pull request merged**